.github/workflows/remove-locked-repo-collaborators.yaml

```diff
@@ -1,6 +1,8 @@
 name: Remove collaborators attached to locked repositories
 on:
   workflow_dispatch:
+  schedule:
+    - cron: "0 4 1 * *"
```

Comment:
It runs a 4:00 am on the 1st of every month. We're not 100% on the value of the script, so minimal runs are preferable to look at the impact more easily retrospectively 👀